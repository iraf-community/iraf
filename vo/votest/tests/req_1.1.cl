#
#  Access to <PARAM> and <INFO> table elements.

# Set the test description string.
votest.descr = "Access to <PARAM> and <INFO> table elements"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"

print ("------------------------------------------------------------------")
print ("Req  1.1:  Users shall be able to access the <PARAM> and <INFO>")
print ("           elements of a VOTable as standard table header information.")
print ("------------------------------------------------------------------")


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
thedit ("data$usno-b.xml", "parnam*,parval*,infnam*,infval*",".",del-,show+)

print ("\nHTTP URI:  " // s1)				# remote http URI
thedit (s1, "parnam*,parval*,infnam*,infval*",".",del-,show+)


print ("\nFile URI:  " // s2)				# file URI
thedit (s2, "parnam*,parval*,infnam*,infval*",".",del-,show+)


print ("\nFile URI:  " // s3)				# file URI
thedit (s3, "parnam*,parval*,infnam*,infval*",".",del-,show+)

