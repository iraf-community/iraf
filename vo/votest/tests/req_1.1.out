------------------------------------------------------------------
Req  1.1:  Users shall be able to access the <PARAM> and <INFO>
           elements of a VOTable as standard table header information.
------------------------------------------------------------------

Logical Path:  data$usno-b.xml
data$usno-b.xml,PARNAM1 = "Field Name" / PARAM name attribute
data$usno-b.xml,PARNAM2 = "Catalogue ID" / PARAM name attribute
data$usno-b.xml,PARNAM3 = "Number of Objects In File" / PARAM name attribute
data$usno-b.xml,PARNAM4 = Equinox / PARAM name attribute
data$usno-b.xml,PARNAM5 = Epoch / PARAM name attribute
data$usno-b.xml,PARNAM6 = "Center RA" / PARAM name attribute
data$usno-b.xml,PARNAM7 = "Center DEC" / PARAM name attribute
data$usno-b.xml,PARNAM8 = "Circle Radius" / PARAM name attribute
data$usno-b.xml,PARNAM9 = "Primary Sort Color" / PARAM name attribute
data$usno-b.xml,PARNAM10 = "Bright Magnitude Limit" / PARAM name attribute
data$usno-b.xml,PARNAM11 = "Faint Magnitude Limit" / PARAM name attribute
data$usno-b.xml,PARNAM12 = "Color0 minus Color1" / PARAM name attribute
data$usno-b.xml,PARNAM13 = "Color Limit: Minimum B-R" / PARAM name attribute
data$usno-b.xml,PARNAM14 = "Color Limit: Maximum B-R" / PARAM name attribute
data$usno-b.xml,PARNAM15 = "Min. Proper Motion" / PARAM name attribute
data$usno-b.xml,PARNAM16 = "Max. Proper Motion" / PARAM name attribute
data$usno-b.xml,PARNAM17 = "Min. Position Error" / PARAM name attribute
data$usno-b.xml,PARNAM18 = "Max. Position Error" / PARAM name attribute
data$usno-b.xml,PARNAM19 = "Min. Proper Motion Error" / PARAM name attribute
data$usno-b.xml,PARNAM20 = "Max. Proper Motion Error" / PARAM name attribute
data$usno-b.xml,PARNAM21 = "Maximum Surface Density" / PARAM name attribute
data$usno-b.xml,PARNAM22 = "Min. Number of Observations" / PARAM name attribute
data$usno-b.xml,PARNAM23 = "Max. Number of Observations" / PARAM name attribute
data$usno-b.xml,PARNAM24 = "Min. Star/Galaxy Sep. index" / PARAM name attribute
data$usno-b.xml,PARNAM25 = "Max. Star/Galaxy Sep. index" / PARAM name attribute
data$usno-b.xml,PARNAM26 = "Lower Bit Mask for Flags" / PARAM name attribute
data$usno-b.xml,PARNAM27 = "Upper Bit Mask for Flags" / PARAM name attribute
data$usno-b.xml,PARNAM28 = "Column Selection Bits" / PARAM name attribute
data$usno-b.xml,PARVAL1 = NoName / PARAM val attribute
data$usno-b.xml,PARVAL2 = USNOB / PARAM val attribute
data$usno-b.xml,PARVAL3 = 65 / PARAM val attribute
data$usno-b.xml,PARVAL4 = 2000.00 / PARAM val attribute
data$usno-b.xml,PARVAL5 = 2000.00 / PARAM val attribute
data$usno-b.xml,PARVAL6 = 202.000000 / PARAM val attribute
data$usno-b.xml,PARVAL7 = 47.000000 / PARAM val attribute
data$usno-b.xml,PARVAL8 = 0.050000 / PARAM val attribute
data$usno-b.xml,PARVAL9 = R2 / PARAM val attribute
data$usno-b.xml,PARVAL10 = 0.00 / PARAM val attribute
data$usno-b.xml,PARVAL11 = 100.00 / PARAM val attribute
data$usno-b.xml,PARVAL12 = B2-R2 / PARAM val attribute
data$usno-b.xml,PARVAL13 = -100.00 / PARAM val attribute
data$usno-b.xml,PARVAL14 = 100.00 / PARAM val attribute
data$usno-b.xml,PARVAL15 = 0.00 / PARAM val attribute
data$usno-b.xml,PARVAL16 = 10000.00 / PARAM val attribute
data$usno-b.xml,PARVAL17 = 0.00 / PARAM val attribute
data$usno-b.xml,PARVAL18 = 10000.00 / PARAM val attribute
data$usno-b.xml,PARVAL19 = 0.00 / PARAM val attribute
data$usno-b.xml,PARVAL20 = 10000.00 / PARAM val attribute
data$usno-b.xml,PARVAL21 = 10000000.00 / PARAM val attribute
data$usno-b.xml,PARVAL22 = 0 / PARAM val attribute
data$usno-b.xml,PARVAL23 = 10 / PARAM val attribute
data$usno-b.xml,PARVAL24 = 0 / PARAM val attribute
data$usno-b.xml,PARVAL25 = 19 / PARAM val attribute
data$usno-b.xml,PARVAL26 = 0 / PARAM val attribute
data$usno-b.xml,PARVAL27 = -1 / PARAM val attribute
data$usno-b.xml,PARVAL28 = 4271 / PARAM val attribute
Warning:  keyword(s) `infnam*' not found.
Warning:  keyword(s) `infval*' not found.

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
http://iraf.noao.edu/votest//usno-b.xml,PARNAM1 = "Field Name" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM2 = "Catalogue ID" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM3 = "Number of Objects In File" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM4 = Equinox / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM5 = Epoch / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM6 = "Center RA" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM7 = "Center DEC" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM8 = "Circle Radius" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM9 = "Primary Sort Color" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM10 = "Bright Magnitude Limit" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM11 = "Faint Magnitude Limit" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM12 = "Color0 minus Color1" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM13 = "Color Limit: Minimum B-R" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM14 = "Color Limit: Maximum B-R" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM15 = "Min. Proper Motion" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM16 = "Max. Proper Motion" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM17 = "Min. Position Error" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM18 = "Max. Position Error" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM19 = "Min. Proper Motion Error" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM20 = "Max. Proper Motion Error" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM21 = "Maximum Surface Density" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM22 = "Min. Number of Observations" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM23 = "Max. Number of Observations" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM24 = "Min. Star/Galaxy Sep. index" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM25 = "Max. Star/Galaxy Sep. index" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM26 = "Lower Bit Mask for Flags" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM27 = "Upper Bit Mask for Flags" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARNAM28 = "Column Selection Bits" / PARAM name attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL1 = NoName / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL2 = USNOB / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL3 = 65 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL4 = 2000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL5 = 2000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL6 = 202.000000 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL7 = 47.000000 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL8 = 0.050000 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL9 = R2 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL10 = 0.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL11 = 100.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL12 = B2-R2 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL13 = -100.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL14 = 100.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL15 = 0.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL16 = 10000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL17 = 0.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL18 = 10000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL19 = 0.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL20 = 10000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL21 = 10000000.00 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL22 = 0 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL23 = 10 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL24 = 0 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL25 = 19 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL26 = 0 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL27 = -1 / PARAM val attribute
http://iraf.noao.edu/votest//usno-b.xml,PARVAL28 = 4271 / PARAM val attribute
Warning:  keyword(s) `infnam*' not found.
Warning:  keyword(s) `infval*' not found.

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM1 = "Field Name" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM2 = "Catalogue ID" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM3 = "Number of Objects In File" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM4 = Equinox / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM5 = Epoch / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM6 = "Center RA" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM7 = "Center DEC" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM8 = "Circle Radius" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM9 = "Primary Sort Color" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM10 = "Bright Magnitude Limit" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM11 = "Faint Magnitude Limit" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM12 = "Color0 minus Color1" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM13 = "Color Limit: Minimum B-R" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM14 = "Color Limit: Maximum B-R" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM15 = "Min. Proper Motion" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM16 = "Max. Proper Motion" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM17 = "Min. Position Error" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM18 = "Max. Position Error" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM19 = "Min. Proper Motion Error" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM20 = "Max. Proper Motion Error" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM21 = "Maximum Surface Density" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM22 = "Min. Number of Observations" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM23 = "Max. Number of Observations" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM24 = "Min. Star/Galaxy Sep. index" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM25 = "Max. Star/Galaxy Sep. index" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM26 = "Lower Bit Mask for Flags" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM27 = "Upper Bit Mask for Flags" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARNAM28 = "Column Selection Bits" / PARAM name attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL1 = NoName / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL2 = USNOB / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL3 = 65 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL4 = 2000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL5 = 2000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL6 = 202.000000 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL7 = 47.000000 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL8 = 0.050000 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL9 = R2 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL10 = 0.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL11 = 100.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL12 = B2-R2 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL13 = -100.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL14 = 100.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL15 = 0.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL16 = 10000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL17 = 0.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL18 = 10000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL19 = 0.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL20 = 10000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL21 = 10000000.00 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL22 = 0 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL23 = 10 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL24 = 0 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL25 = 19 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL26 = 0 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL27 = -1 / PARAM val attribute
file:///iraf/iraf/vo/votest/data//usno-b.xml,PARVAL28 = 4271 / PARAM val attribute
Warning:  keyword(s) `infnam*' not found.
Warning:  keyword(s) `infval*' not found.

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM1 = "Field Name" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM2 = "Catalogue ID" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM3 = "Number of Objects In File" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM4 = Equinox / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM5 = Epoch / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM6 = "Center RA" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM7 = "Center DEC" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM8 = "Circle Radius" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM9 = "Primary Sort Color" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM10 = "Bright Magnitude Limit" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM11 = "Faint Magnitude Limit" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM12 = "Color0 minus Color1" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM13 = "Color Limit: Minimum B-R" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM14 = "Color Limit: Maximum B-R" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM15 = "Min. Proper Motion" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM16 = "Max. Proper Motion" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM17 = "Min. Position Error" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM18 = "Max. Position Error" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM19 = "Min. Proper Motion Error" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM20 = "Max. Proper Motion Error" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM21 = "Maximum Surface Density" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM22 = "Min. Number of Observations" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM23 = "Max. Number of Observations" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM24 = "Min. Star/Galaxy Sep. index" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM25 = "Max. Star/Galaxy Sep. index" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM26 = "Lower Bit Mask for Flags" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM27 = "Upper Bit Mask for Flags" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARNAM28 = "Column Selection Bits" / PARAM name attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL1 = NoName / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL2 = USNOB / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL3 = 65 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL4 = 2000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL5 = 2000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL6 = 202.000000 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL7 = 47.000000 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL8 = 0.050000 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL9 = R2 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL10 = 0.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL11 = 100.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL12 = B2-R2 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL13 = -100.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL14 = 100.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL15 = 0.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL16 = 10000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL17 = 0.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL18 = 10000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL19 = 0.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL20 = 10000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL21 = 10000000.00 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL22 = 0 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL23 = 10 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL24 = 0 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL25 = 19 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL26 = 0 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL27 = -1 / PARAM val attribute
file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml,PARVAL28 = 4271 / PARAM val attribute
Warning:  keyword(s) `infnam*' not found.
Warning:  keyword(s) `infval*' not found.
